pages/checkout/index.js

```diff
@@ -12,6 +12,14 @@ import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
 import { useRouter } from 'next/router';
 
+import Navigation from '../../components/Navigation';
```

Comment:
It's generally a good practice to add relative imports (your own components) after absolute imports (material-ui components) to avoid some css cascade issues